python/matplotlib.ipynb

```diff
@@ -869,7 +1147,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.6"
+   "version": "3.11.9"
```

Comment:
Did we remove the metadata cleaner from the pre-commit hooks?
Also do we show `python 3.11` or `3.12` currently in the workshop?

Reply:
And we should use Python 3.12. I updated the environment in #422